mmv1/third_party/terraform/website/docs/r/storage_transfer_job.html.markdown

```diff
@@ -112,6 +112,8 @@ resource "google_storage_transfer_job" "s3-bucket-nightly-backup" {
 
 The following arguments are supported:
 
+* `name` - (Optional) The name of the Transfer Job. If unspecified, the provider will assign a random unique name with `transferJobs/{{name}}` format, where `name` is a numeric value.
```

Comment:
Can you please provide more information about the format of `name`?  You can copy the information from API doc https://cloud.google.com/storage-transfer/docs/reference/rest/v1/transferJobs.

```
This name must start with "transferJobs/" prefix and end with a letter or a number, and should be no more than 128 characters. For transfers involving PosixFilesystem, this name must start with transferJobs/OPI specifically. For all other transfer types, this name must not start with transferJobs/OPI.

Non-PosixFilesystem example: "transferJobs/^(?!OPI)[A-Za-z0-9-._~]*[A-Za-z0-9]$"

PosixFilesystem example: "transferJobs/OPI^[A-Za-z0-9-._~]*[A-Za-z0-9]$"
```

Reply:
Yes, I added the information and condensed it into one line.